### PR TITLE
[docs] Update Debezium connector docs.

### PR DESCRIPTION
**Pull request by ryzhyk**
* Consolidate Postgres and MySQL Debezium docs in one document.
* Don't document database and Kafka connect configuration in detail. These are well documented elsewhere and we don't need to duplicate these docs.
* Document Avro support.

While working on the docs, I also removed unnecessary configs from the Debezium demo.

### CODE REVIEW

File: docs/connectors/sources/debezium.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+
+:::note
+This page describes configuration options specific to the Debezium source connector.
+See [top-level connector documentation](/docs/connectors/) for general information
+about configuring input and output connectors.
+:::
+
+Debezium is a widely-used **Change Data Capture** (CDC) technology that streams real-time changes from databases such
+as PostgreSQL, MySQL, and Oracle to Kafka topics. Feldera can consume these change streams as inputs. We support
+Debezium streams encoded in both [JSON](/docs/formats/json.md) and [Avro](/docs/formats/avro.md) formats. Synchronizing
```
Comment (gz) [inline docs/connectors/sources/debezium.md:11]: btw maybe we can have  a table in the higher level connectors page that shows which connector supports which format

**Comment by ryzhyk** [comment]
test comment

File: docs/connectors/sources/debezium.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+            "topic.prefix": "[DATABASE SERVER NAME]",
+            "schema.history.internal.kafka.topic": "schema-changes.[DATABASE SERVER NAME].internal",
+            "schema.history.internal.kafka.bootstrap.servers": "[KAFKA HOSTNAME:PORT]",
+            "include.schema.changes": "true",
+            "decimal.handling.mode": "string",
+        }
+    }'
+```
+
+:::note
```
Comment (gz) [inline docs/connectors/sources/debezium.md:122]:
```suggestion
:::tip
```

File: docs/connectors/sources/debezium.md [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+
+## Step 2: Create Kafka Connect input connector
+
+Debezium is built on top of
+[Kafka Connect](https://docs.confluent.io/platform/current/connect/index.html).  You will need
+to install Kafka Connect along with a Debezium plugin for your database.
+Next you will need to use the Kafka Connect REST API to create a source connector to stream
+changes from the database change log to a Kafka topic.
+
+Refer to the Debezium documentation to configure the connector according to your requirements,
```
Comment (gz) [inline docs/connectors/sources/debezium.md:32]: hyperlink the "Debezium documentation"

File: docs/connectors/sources/debezium.md [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+
+:::note
+Refer to the [secret management](../../enterprise/kubernetes-guides/secret-management) guide
+to externalize secrets such as DBMS passwords via Kubernetes.
+:::
+
+## Step 3: Create Feldera input connector
+
+Configure an input connector for each Feldera SQL table that must ingest changes from Debezium.
+Use the `kafka_input` transport with either `json` or `avro` format. Debezium automatically
```
Comment (gz) [inline docs/connectors/sources/debezium.md:130]:
```suggestion
Use the `kafka_input` transport with using either `json` or `avro` as the format. Debezium automatically
```

File: docs/connectors/sources/debezium.md [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+creates a Kafka topic for each database table.
+
+### JSON
+
+When using JSON encoding, make sure to set the following connector properties:
+
+* `"update_format": "debezium"`
+* `"json_flavor"` depending on the:
```
Comment (gz) [inline docs/connectors/sources/debezium.md:138]:
```suggestion
* `"json_flavor"` depending on the Database:
```

File: docs/connectors/sources/debezium.md [hunk 6]
```diff
@@ -0,0 +1,226 @@
...
+  * For MySQL and MariaDB, set `"json_flavor": "debezium_mysql"`
+  * For all other databases set `"json_flavor": "debezium_postgres"`
+
+Configure a Feldera connector to ingest changes from a Postgres DB via a JSON-encoded Kafka topics:
+
+```sql
+CREATE TABLE my_table (
+   ... -- columns omitted
```
Comment (gz) [inline docs/connectors/sources/debezium.md:146]:
```suggestion
   example_field INT
```

which has the nice effect that this compiles out of the box (I want to make a script that checks all our examples in docs against the sandbox)

File: docs/connectors/sources/debezium.md [hunk 7]
```diff
@@ -0,0 +1,226 @@
...
+      }
+  }]'
+)
+```
+
+Configure a Feldera connector to ingest changes from a Postgres DB via a JSON-encoded Kafka topics:
+
+```sql
+CREATE TABLE my_table (
+   ... -- columns omitted
```
Comment (gz) [inline docs/connectors/sources/debezium.md:172]: same here

**Review by gz — APPROVED** [review]

**Pull request merged**